tools/regression_test.sh

```diff
@@ -127,7 +127,7 @@ function main {
       tmp=$DB_PATH
       DB_PATH=$ORIGIN_PATH
       test_remote "test -d $DB_PATH"
-      if [[ $? -ne 0 ]] || [[ $(run_remote 'echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))') -gt "604800" ]]; then
+      if [[ $? -ne 0 ]] || [[ $(run_remote \''echo $(( $(date +"%s") - $(stat -c "%Y" '"$DB_PATH"') ))'\') -gt "604800" ]]; then
```

Comment:
i think you want this in test_remote. Otherwise it'll work only conditionally depending on $REMOTE_USER_AT_HOST.